apisix/plugins/file-logger.lua

```diff
@@ -0,0 +1,102 @@
+--
+-- Licensed to the Apache Software Foundation (ASF) under one or more
+-- contributor license agreements.  See the NOTICE file distributed with
+-- this work for additional information regarding copyright ownership.
+-- The ASF licenses this file to You under the Apache License, Version 2.0
+-- (the "License"); you may not use this file except in compliance with
+-- the License.  You may obtain a copy of the License at
+--
+--     http://www.apache.org/licenses/LICENSE-2.0
+--
+-- Unless required by applicable law or agreed to in writing, software
+-- distributed under the License is distributed on an "AS IS" BASIS,
+-- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+-- See the License for the specific language governing permissions and
+-- limitations under the License.
+--
+local log_util     =   require("apisix.utils.log-util")
+local core         =   require("apisix.core")
+local plugin       =   require("apisix.plugin")
+local ngx          =   ngx
+local io_open      =   io.open
+
+
+local plugin_name  =   "file-logger"
+local file_descriptors = {}
+
+
+local schema = {
+    type = "object",
+    properties = {
+        path = {
+            type = "string",
+            match = [[^[^*&%%\`]+$]],
```

Comment:
The syntax is still incorrect. See https://github.com/apache/apisix/pull/5831#discussion_r776539246